saw-core/src/Verifier/SAW/Simulator/Value.hs

```diff
@@ -343,6 +345,7 @@ asFirstOrderTypeTValue v =
     VSort{}     -> Nothing
     VRecursorType{} -> Nothing
     VTyTerm{}   -> Nothing
+    VStringType -> Nothing
```

Comment:
I'm not sure about this case. Does it make sense to add a `FOTString` constructor? I'm unclear what the consequences of that would be.